cobc/parser.y

```diff
@@ -12101,6 +12101,11 @@ display_body:
 	cb_emit_command_line ($1);
   }
 | screen_or_device_display _common_exception_phrases
+| _with CONVERSION screen_or_device_display _common_exception_phrases
+  {
+	check_repeated ("CONVERSION", SYN_CLAUSE_8, &check_duplicate);
```

Comment:
There is no way to have CONVERSION repeated here, is it? If it isn't drop the check.
Explanation: it is used in cases where a single statement can have multiple modifiers - to ensure no modifier is used more than once and to ensure no conflicting modifier is used.

Reply:
Indeed. Fixed. ;)